aws-predictions-tensorflow/src/main/java/com/amplifyframework/predictions/tensorflow/service/TensorFlowTextClassificationService.java

```diff
@@ -202,9 +169,9 @@ Sentiment fetchSentiment(String text) throws PredictionsException {
      */
     @WorkerThread
     synchronized void release() {
-        for (Loadable<?, PredictionsException> asset : assets) {
-            asset.unload();
-        }
+        interpreter.unload();
+        dictionary.unload();
+        labels.unload();
```

Comment:
The compound structure extends `Latch`, so the focus is on that class is on "executing a latched task" which just happened to be loading in this case. I figured it would be strange to add `unload()` method to it.